GNUmakefile.in

```diff
@@ -18,6 +18,9 @@ ABI = @ABI@
 HPCGAP = @HPCGAP@
 ADDGUARDS2 = @ADDGUARDS2@
 
+# LibGAP specific
+LIBGAP = @LIBGAP@
```

Comment:
I would recommend renaming this to BUILD_LIBGAP to make it clearer what this does. Also fits with BUILD_LIBATOMIC_OPS, BUILD_GMP, BUILD_BOEHM_GC